src/slidepane/styles/slidePane.m.css

```diff
@@ -8,17 +8,16 @@
 	left: 0;
 	width: 100%;
 	height: 100%;
-	z-index: 1;
 }
 
 .underlayVisible {
 	background: var(--underlay-background);
+	z-index: var(--zindex-dialog);
```

Comment:
I see, so it looks like you're using `--zindex-dialog` for underlays, and then adding one for the content itself. For some reason that seems odd to me, maybe it's just a matter of variable naming.

Reply:
I think technically they could both be `--zindex-dialog`, since the dialog is after the underlay in the dom. I'd be fine with that if you are, although the `+ 1` is a little nice in that it makes the layering explicit.

Reply:
I'm fine with everything being explicit.